README.md

```diff
@@ -38,14 +38,12 @@ To perform manual releases, you will need:
 4. Run `bundle exec rails server` to launch the app on http://localhost:3000
 5. Run `./bin/webpack-dev-server` in a separate shell for faster compilation of assets
 
-### Setting up some development data
+### Setting up local development data
 
-There are several tasks you can run to get schools and responsible bodies:
+These are the tasks that you will need to run to set up your local db:
 
-- Run `bundle exec rake import:responsible_bodies` to import local authorities and trusts
-- Run `bundle exec rake import:schools` to import schools into staging area
-- Then open up a rails console with `bundle exec rails c`
-- Run `service = SchoolUpdateService.new; DataStage::School.all.each { |staged| puts "importing #{staged.urn}..."; service.create_school!(staged) }` to convert staged schools into schools
+- Run `bundle exec rails db:seed` to insert mobile network operators
```

Comment:
I'm in two minds as to whether this should still be a seed or whether we should move it into the `import` task namespace. What do others think?

Reply:
Feels like we should be consistent and either make them all seeds or all rake tasks. I don't know if there is a reason this needs to be in the seeds file rather than a rake task, maybe someone else knows?